src/pubsub.rs

```diff
@@ -470,7 +482,12 @@ impl AccountUpdateManager {
             (InflightRequest::Sub(sub, commitment), Instant::now()),
         );
         self.subs.insert((sub, commitment), Meta::new());
-        self.send(&request)?;
+        match self.send(&request) {
```

Comment:
what's the point of this change?

Reply:
In first iteration with total sub count limitation it was used to decrement that count, I forgot to revert it back, will do

Reply:
reverted back